software/ghstats.yml

```
@@ -0,0 +1,12 @@
+name: ghstats
+website_url: https://github.com/vladkens/ghstats
+source_code_url: https://github.com/vladkens/ghstats
+description: Dashboard for tracking GitHub repos traffic history longer than 14 days.
+licenses:
+  - MIT
+platforms:
+  - Docker
+tags:
+  - Personal Dashboards
+  - Analytics
```

```suggestion
tags:
  - Analytics
```

I know the https://awesome-selfhosted.net/tags/personal-dashboards.html section is not clearly defined, but ghstats would look quite out-of-place there.

---

@vladkens Please see the above recommendation.